lib/doorkeeper/openid_connect/version.rb

```diff
@@ -1,5 +1,5 @@
 module Doorkeeper
   module OpenidConnect
-    VERSION = '1.1.2'.freeze
+    VERSION = '1.2.0'.freeze
```

Comment:
no, it isn't something we can submit, it needs quite a bit more work to make it so we can send upstream. Too much hard-coded, i.e. support for EC and the venuenext.net domain.